constructor/sh/construct.yaml

```diff
@@ -0,0 +1,28 @@
+name: omnisci
+version: 0.0.2
+company: OmniSci, Inc.
+
+keep_pkgs: True
+attempt_hardlinks: True
+ignore_duplicate_files: True
+install_in_dependency_order: True
+installer_type: sh  # [osx]
```

Comment:
it is just a copy from pkg/construct.yaml ... the change here is just the installer_type sh